man/systemd.target.xml

```diff
@@ -95,7 +95,8 @@
       <listitem><para>Target units will automatically complement all
       configured dependencies of type <varname>Wants=</varname> or
       <varname>Requires=</varname> with dependencies of type
-      <varname>After=</varname>. Note that <varname>Wants=</varname> or
+      <varname>After=</varname> unless <varname>DefaultDependencies=no</varname>
+      is not set in the specified units. Note that <varname>Wants=</varname> or
```

Comment:
I think proper English is "unless is set". Here you effectively have double negation, so the effect - `After` is not added   if `DefaultDependecies=no` is **not** present.

Reply:
Ahh... Thanks. I will update.

Reply:
Updated.